src/constraints/vectoraffine.jl

```diff
@@ -76,6 +76,62 @@ function MOI.modify!(model::LinQuadOptimizer, index::VLCI{<: VecLinSets},
     end
 end
 
+_constant(f::Linear) = f.constant
+_constant(f::VecLin) = f.constants
```

Comment:
Are there MOIU functions for this?

Reply:
There are, but they're likewise `_`-prefixed so I didn't want to use them externally. `MOIU` does have `constant()` but it unnecessarily allocates a vector for scalar functions.